.github/workflows/unit_tests.yml

```diff
@@ -19,8 +19,8 @@ jobs:
           git submodule init
           git submodule update
       - name: Cmake Makefiles
-        run: cmake -B ./build -G 'Unix Makefiles'
+        run: USE_GPIO=1 cmake -B ./build -G 'Unix Makefiles'
       - name: Run make
-        run: cmake --build ./build --target tests
+        run: USE_GPIO=1 cmake --build ./build --target tests
```

Comment:
I forget why we can't use the script on GitHub actions, is it due to ninja not working? @jonapap

Reply:
I think it was because we just want to build the `tests` target. However, now our script support specifying the target, so we could use the script.

Oh, and I personally believe we should remove the guards and allow the `.cpp` files to compile even if the preprocessor variable is not set. This way you don't have to add `USE_GPIO = 1` here.

Reply:
The only disadvantage I found with removing guards is that it requires removing guards from `sensorsData` which makes it slightly less type-safe.